app/assets/javascripts/actions/index.coffee

```diff
@@ -0,0 +1,37 @@
+Screensmart.Actions =
+  addMessage: (message) ->
+    type: 'ADD_MESSAGE'
+    message: message
+
+  setAnswer: (key, value) ->
+    (dispatch) =>
+      dispatch @_setAnswer(key, value)
+      dispatch @updateResponse()
+
+  _setAnswer: (key, value) ->
+    type: 'SET_ANSWER'
+    key: key
+    value: value
+
+  updateResponse: ->
+    (dispatch, getState) ->
+      response = getState().response
+      syncResponse(response).then (data) ->
+        dispatch Screensmart.Actions.receiveResponseUpdate(data)
+
+  setResponseLoading: ->
+    type: 'SET_RESPONSE_LOADING'
+
+  receiveResponseUpdate: (data) ->
+    type: 'RECEIVE_RESPONSE_UPDATE'
+    response: data.response
+
+syncResponse = (response) ->
+  $.ajax '/responses',
+    method: 'POST'
+    dataType: 'json'
+    contentType: 'application/json'
+    data: JSON.stringify
+      response: response
+    headers:
+      'X-CSRF-Token': window.csrfToken
```

Comment:
Er zit nu geen rails.js in het project, dus volgens mij moet je dit zelf opgeven.

Ik zal het nog wel verplaatsen naar de globale jQuery configuratie, want het CSRF token is nodig voor elk POST verzoek.